config/jobs/image-pushing/README.md

```diff
@@ -160,7 +160,7 @@ If you need the `.git` directory to be uploaded to your build environment:
 - add an empty `.gcloudignore` file to your repository. This will override the
   [default values][gcloudignore].
 
-[gcr instructions]: https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/README.md
+[gcr instructions]: https://github.com/kubernetes/k8s.io/blob/main/registry.k8s.io/README.md
```

Comment:
The instructions are still at https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/README.md

Can we please double check that changes are valid instead of mass substituting?

For all references to using/pulling an image, registry.k8s.io contains all images from k8s.gcr.io

For other contexts, they may not be interchangeable.